src/indicators/flooding/glofas/utils.py

```diff
@@ -303,87 +307,84 @@ def get_groups_above_threshold(
     return [group for group in groups if group[1] - group[0] >= min_duration]
 
 
-def get_glofas_activations(da_glofas, thresh, ndays):
-    vals = da_glofas.values
-    groups = get_groups_above_threshold(vals, thresh, ndays)
-    df_glofas_act = pd.DataFrame(groups, columns=["start_index", "end_index"])
-    df_glofas_act["num_days"] = (
-        df_glofas_act["end_index"] - df_glofas_act["start_index"]
-    )
-    df_glofas_act["start_date"] = df_glofas_act["start_index"].apply(
-        lambda x: da_glofas.time[x].values
-    )
-    df_glofas_act["end_date"] = df_glofas_act["end_index"].apply(
-        lambda x: da_glofas.time[x].values
+def get_dates_list_from_dataset(
```

Comment:
Since we talked about it today, could we add a docstring to this function? :)

Reply:
Good call, done!